NVIDIA-Driver/install.bash

```diff
@@ -54,6 +55,56 @@ Section "Files"
         ModulePath      "/opt/nvidia/lib64/xorg/modules"
 EndSection
 EOF
+cat <<EOF | sudo tee /etc/X11/xorg.conf.d/10-nvidia-drm-outputclass.conf > /dev/null
+# This xorg.conf.d configuration snippet configures the X server to
+# automatically load the nvidia X driver when it detects a device driven by the
+# nvidia-drm.ko kernel module.  Please note that this only works on Linux kernels
+# version 3.9 or higher with CONFIG_DRM enabled, and only if the nvidia-drm.ko
+# kernel module is loaded before the X server is started.
+Section "OutputClass"
+    Identifier  "intel"
+    MatchDriver "i915"
+    Driver      "modesetting"
+EndSection 
+
+Section "OutputClass"
+    Identifier      "nvidia"
+    MatchDriver     "nvidia-drm"
+    Driver          "nvidia"
+    Option          "AllowEmptyInitialConfiguration"
+    Option          "PrimaryGPU" "yes"
+    ModulePath      "/opt/nvidia/lib64/xorg/modules"
+    ModulePath      "/usr/lib64/xorg/modules"
+EndSection
+EOF
+cat <<EOF | sudo tee /etc/X11/xorg.conf.d/90-mwhd.conf > /dev/null
+Section "Module"
+	Load "modesetting"
+EndSection
+
+Section "Device"
+	Identifier "nvidia"
+	Driver "nvidia"
+	BusID "PCI:1:0:0"
+	Option "AllowEmptyInitialConfiguration"
+EndSection
+EOF
```

Comment:
Any references?

Reply:
https://wiki.archlinux.org/index.php/NVIDIA_Optimus#GDM

Reply:
Line 58 to 79 LGTM. But  Line 76 to 77 may be redundant given line 52 to 55.

---

Line 88 could be problematic. Suppose the GPU is not on PCI 1, or there are multiple GPUs, this won't work properly.